src/datachain/lib/dc.py

```diff
@@ -1086,7 +1086,7 @@ def mutate(self, **kwargs) -> "Self":
         Example:
         ```py
          dc.mutate(
-                newkey=Column("oldkey")
+            newkey=Column("oldkey")
```

Comment:
should we expand docs here a bit?

Reply:
I think we have explanation here ([above](https://github.com/iterative/datachain/blob/main/src/datachain/lib/dc.py#L1082-L1090)), I just fix the indentation.

Reply:
I mean - do we need to add something about `over` and `window` to docs somewhere?

Reply:
Oh, that's a good question. Definitely, but if we'll add it here, in `mutate` function docstrings it can become pretty big. Do you have an idea on where to put these docs? Or it is ok to put it here?

Reply:
I've updated `mutate` function docstring, please, take a look?